DuckDuckGo/YoutubePlayer/DuckPlayer.swift

```diff
@@ -178,6 +189,16 @@ final class DuckPlayer {
         isFeatureEnabled = privacyConfigurationManager.privacyConfig.isEnabled(featureKey: .duckPlayer)
         isPiPFeatureEnabled = privacyConfigurationManager.privacyConfig.isSubfeatureEnabled(DuckPlayerSubfeature.pip)
         isAutoplayFeatureEnabled = privacyConfigurationManager.privacyConfig.isSubfeatureEnabled(DuckPlayerSubfeature.autoplay)
+        isCustomErrorFeatureEnabled =
+            privacyConfigurationManager.privacyConfig.isSubfeatureEnabled(DuckPlayerSubfeature.customError)
+
+        // TODO: Check if this logic can be moved into privacyConfig
```

Comment:
`PrivacyConfigurationData` has `FeatureSettings = [String: Any]` and `SubfeatureSettings = String` even though they are the same data type in the privacy config schema.

Is `SubfeatureSettings` just a stub that should be refactored to return `[String: Any]` like `FeatureSettings`, which would make the code block below much smaller?